### ISSUE TITLE
Multichain support 

### ISSUE THREAD

**Issue by StatusSceptre** [status-im/status#8]
L2 networks are gaining adoption (testnet and mainnet). This issue captures the user story and related actions to support use of dapps on these networks.

_(Out of scope for this issue is what solution Status would use for payment network and SNT utility)_

#### User story
As a user I want to use Status on different networks, so that I can continue using dapps that offer services off Ethereum Mainnet (e.g. Uniswap, Synthetix)

#### Actions
- [ ] Support multiple child and side chains by default
_P0 Optimism
Px Arbitrum, Matterlabs, Starkware, Fuel, Avalanche_
- [ ] Review manual entry network connection for any updates needed
- [ ] Define UI changes to connect, transact, view transactions by chain
- [ ] Support connecting to chains in parallel
- [ ] Middle layer to support tx between networks

Sorry, wrong account (created by @hesterbruikman)
